packages/spiderfier/marker-spider.module.ts

```diff
@@ -0,0 +1,11 @@
+import {NgModule} from '@angular/core';
+import {AgmCoreModule} from '@agm/core';
+import {AgmMarkerSpider} from './directives/marker-spider';
+
+@NgModule({
+  imports: [AgmCoreModule],
```

Comment:
we don't need to import core here, as no component/directive/provider is used in the code

Reply:
Will update this